images/base/files/usr/local/bin/entrypoint

```diff
@@ -27,6 +27,11 @@ if grep -Eqv "0[[:space:]]+0[[:space:]]+4294967295" /proc/self/uid_map; then
   echo 'INFO: running in a user namespace (experimental)'
 fi
 
+grep_allow_nomatch() {
+  # grep exits 0 on match, 1 on no match, 2 on error
+  grep "$@" || [[ $? == 1 ]]
```

Comment:
TODO: We could consider using a less terse version and retain the exact exit code.
right now that's not important and I'd rather not have to rebase and re-build https://github.com/kubernetes-sigs/kind/pull/2737 again unless there's a seriours error here.

We probably should follow-up with preserving the exact status for future usage.

Or just switch to pure awk.

Reply:
Yeah, it seems that awk would look nicer. Something like

```bash
unsupported_cgroups=$(findmnt -lun -o source,target -t cgroup | awk '!/'"${current_cgroup}"'/ {print $2}')
```

Reply:
It feels slightly wrong that we're regex matching in either case instead of substring literal but :this-is-fine:
I suppose we could regex escape it with some more bash 🙈 

It looks like if we stuck with grep we could use `-F` / `--fix-strings`.

Reply:
For `awk`, one can use something `index(string, substring) == -1`. Some implementations treat substr as a regexp, but GNU awk does a fixed substring match.